eng/pipelines/common/global-build-job.yml

```diff
@@ -27,8 +27,7 @@ parameters:
   helixQueues: ''
   enablePublishTestResults: false
   testResultsFormat: ''
-  extraStepsTemplate: ''
-  extraStepsParameters: {}
+  postBuildSteps: []
```

Comment:
Should the name have a `Template` suffix like we have for `jobTemplate` in other places.

Reply:
I thought about it, but since we allow both template and non-template options, I wanted to match something closer to AzDO's naming. For plain AzDO YAML (without the absurd number of layers we have), the name of the parent is just `steps` can can be a template or a non-template.